include/deal.II/distributed/field_transfer.h

```diff
@@ -52,17 +52,11 @@ namespace parallel
          * Constructor.
          *
          * @param[in] dof_handler The DoFHandler on which all the operations
-         * will happen. At the time when this constructor is call, the
-         * DoFHandler still points to the Triangulation before the refinement in
-         * question happens.
+         * will happen. This constructor must be called before the underlying
+         * Triangulation is coarsened/refined.
          */
         FieldTransfer(const DoFHandler<dim, spacedim> &dof_handler);
 
-        /**
-         * Destructor.
-         */
-        ~FieldTransfer() = default;
```

Comment:
Why did you remove this?

Reply:
It isn't doing anything. So I follow the rule of zero.